http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -1435,8 +1435,10 @@ class LeoPubsubMessageSubscriber[F[_]](
           case e =>
             for {
               _ <- appQuery.updateStatus(msg.appId, AppStatus.Running).transaction
+              errorContext = s"Error updating Azure app with id ${msg.appId.id} and cloudContext ${msg.cloudContext.asString}"
+              _ = logger.warn(ctx.loggingCtx, e)(errorContext)
```

Comment:
this won't log anything, it needs to be `_ <- logger.warn(ctx.loggingCtx, e)(errorContext)`.

Reply:
since it returns an `F[]`, the log statement won't be executed unless its in the flatmap. It's similar to if you have a database query and are trying to  make it an atomic set of instructions, you need to flatmap over each database transaction `F`. Here an `F[]` is declared but it is never executed, just assigned and left dangling

Reply:
Fascinating, thanks! Apparently my preconception ("and presumably not a F[] ...") was completely wrong

Reply:
Now changed